Kickstarter-iOS/Views/Controllers/SearchViewController.swift

```diff
@@ -16,13 +16,26 @@ internal final class SearchViewController: UITableViewController {
   @IBOutlet fileprivate weak var searchStackView: UIStackView!
   @IBOutlet fileprivate weak var searchTextField: UITextField!
 
+  private let backgroundView = UIView()
+  private let popularLoaderIndicator = UIActivityIndicatorView()
+  private let searchLoaderIndicator = UIActivityIndicatorView()
```

Comment:
@mbrandonw for context here, this was a maggie choice to have a loader in 2 places. considering they are 2 different api requests, it seemed easiest and less error-prone to use 2 loaders vs play with constraints based on which request was being made. i agree the table header is not ideal, especially because there can be a delay where it shifts content upwards when it is set to nil.